src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/InputNoLineWrapBad.java

```diff
@@ -1,15 +1,17 @@
 package com.puppycrawl.tools.
     checkstyle.checks.whitespace;
 
-import com.puppycrawl.tools.checkstyle.TreeWalker;
+import static java.math.
+	BigInteger.ZERO;
 
-import javax.accessibility.
-    AccessibleAttributeSequence;
+import com.puppycrawl.tools.checkstyle.TreeWalker;
 import java.util.concurrent.atomic.AtomicIntegerFieldUpdater;
+import javax.accessibility.
+		AccessibleAttributeSequence;
```

Comment:
@kukreja-vikramaditya 
I do not your problem to understand too. Checkstyle does not validate test input files.
In your case, for example, for InputNoLineWrapGood this works fine:
```java
import static java.math.BigInteger.ZERO;

import com.puppycrawl.tools.checkstyle.TreeWalker;

import javax.accessibility.AccessibleAttributeSequence;
import java.util.concurrent.atomic.AtomicIntegerFieldUpdater;

public class InputNoLineWrapGood {
```

As you can see, I just added new static import and did not touch any other imports in the file.